blocks.go

```diff
@@ -117,24 +117,28 @@ func FillBlockContent(block *models.Block, blockRequest *notespb.Block) error {
 func FillContentFromModelToApi(blockRequest *models.BlockWithIndex, contentType uint32, blockApi *notespb.Block) error {
```

Comment:
I think we should be really careful with variable naming.

Here `blockRequest` is inappropriate, it's not really a "request" object, it's just a model structure right?

`contentType` is an enum? There are magic numbers everywhere. I don't understand what the switch statement does 🤔

Also `blockApi`, it's not an API? It's just a block structure right? It should be named just `block` or the like.

I really think we should be careful because it makes code hard to read + maintain.

Reply:
I'm tagging @gabriel654165 because it's likely that you wrote that.

Reply:
yeeees okay i'm going to fix this in pr #28, anyway edouard must merge this PR after mine because of the missing validators but thanks @rojas-diego

Reply:
> I'm tagging @gabriel654165 because it's likely that you wrote that.

Fixed it in #28